pytradfri/device/socket_control.py

```diff
@@ -1,34 +1,32 @@
 """Class to control the sockets."""
 from __future__ import annotations
 
-from typing import Any
-
 from ..command import Command
 from ..const import ATTR_DEVICE_STATE, ATTR_SWITCH_PLUG
 from .base_controller import BaseController
-from .socket import Socket
+from .socket import Socket, SocketResponse
 
 
 class SocketControl(BaseController):
     """Class to control the sockets."""
 
     @property
-    def raw(self) -> list[dict[str, Any]]:
+    def raw(self) -> list[SocketResponse]:
         """Return raw data that it represents."""
         socket_control_response = self._device.raw.socket_control
         assert socket_control_response is not None
         return socket_control_response
 
     @property
-    def sockets(self):
+    def sockets(self) -> list[Socket]:
         """Return socket objects of the socket control."""
         return [Socket(self._device, i) for i in range(len(self.raw))]
 
-    def set_state(self, state, *, index=0):
+    def set_state(self, state: int, *, index=0) -> Command:
         """Set state of a socket."""
-        return self.set_values({ATTR_DEVICE_STATE: int(state)}, index=index)
+        return self.set_values({ATTR_DEVICE_STATE: state}, index=index)
 
-    def set_values(self, values, *, index=0):
+    def set_values(self, values: dict[str, int], *, index) -> Command:
```

Comment:
Should we keep the default index? Otherwise it's a breaking change.

Reply:
Yes, that was an unintended removal.